src/components/CollectionList/CollectionList.tsx

```diff
@@ -13,9 +13,15 @@ export const CollectionList: React.FC<{ collections: Opds2Feed }> = ({
           <SimpleGrid gap="l" columns={4}>
             {collections.groups.map((collection, index) => {
               if (index < 8)
```

Comment:
Let's turn this magic number 8 into a named constant so we know why it is specifically set to 8